src/index.html

```diff
@@ -5,7 +5,12 @@
     <title>InfoVis</title>
   </head>
   <body>
-      <script src="main.js"></script>
-      <div id="slope"></div>
+    <div id="slope"></div>
+    <h1>Attack Bonus of Civilization Units</h1>
+    <p>The heatmap below shows the attack bonuses of a selected allied civilization against the units of a rival civilization.</p>
```

Comment:
Which civilization is which is still a bit confusing. Perhaps you could add a label to each axis. Also, maybe it would be better to call them simply `Attacker` and `Defender`? Since you would still get bonus damage against yourself if fighting against the other civilization.

Reply:
Sure I wasn't confident about the names either. I think those options are more sutiable as well.